test/SurveyDesign.jl

```diff
@@ -14,6 +14,25 @@
     # TODO: needs change; this works but isn't what the user is expecting
     srs_prob = SimpleRandomSample(apisrs; probs = fill(0.3, size(apisrs_original, 1)))
     @test srs_prob.data.probs[1] == 0.3
+    @test srs_prob.data.weights == 1 ./ srs_prob.data.probs
+
+    srs = SimpleRandomSample(apisrs, ignorefpc = true, weights = :pw )
+    @test srs.data.probs == 1 ./ srs.data.weights
+    
+    srs = SimpleRandomSample(apisrs, ignorefpc = false, weights = :pw)
+    @test srs.data.probs == 1 ./ srs.data.weights
+
+    srs_weights = SimpleRandomSample(apisrs, ignorefpc = false, weights = :fpc)
+    @test_throws srs_weights = SimpleRandomSample(apisrs, ignorefpc = false, weights = :stype)
+    srs_w_p = SimpleRandomSample(apisrs, ignorefpc = false, weights = :fpc, probs = fill(0.3, size(apisrs_original, 1)))
+
+    srs = SimpleRandomSample(apisrs, ignorefpc = true, probs = 1 ./ apisrs.pw )
+    @test srs.data.probs == 1 ./ srs.data.weights
```

Comment:
This test fails and it should indeed fail because, I believe the probability weights should be frequency weights divided by population size. In this case,  `probs` should be given `apisrs.pw ./ apisrs.fpc` and we should check that `srs.data.weights == srs.data.probs .* srs.popsize`.

Reply:
Also, remove the space before the last paranthesis on line 29.